### ISSUE TITLE
Packages

### ISSUE THREAD

**Issue by msusta** [sensorian/sensorian-imagebuilder#2]
I'm not a huge fan of long bash scripts to install programs into systems.
Have you ever thought of pushing this as packages directly in raspbian? Was there any effort done on that front?

**Comment by Gunsmithy** [comment]
I don't believe anyone involved in the project, myself included, are MOTUs, so there hasn't been any attempt to push this as a package as far as I'm aware. I personally try to refine the build script as much as possible, but I would definitely considering packaging it at least for an unofficial repository. I'm not entirely familiar with the process, but I would like to do the same for the related projects as well including support for pip installs.

This repository itself, however, is primarily for building an image from a fresh Raspbian install. [The official firmware repository](https://github.com/sensorian/sensorian-firmware/) would be the place to go for simply adding it to an existing system, but the setup procedure there is definitely lacking I know. It is also something I am trying to improve in a Fixes branch some day.

The fastest solution is definitely to download the pre-built images which were made from this repository to begin with. Outdated ones can be found [on the Sensorian website](http://sensorian.io/downloads/), but I'll be keeping current ones on a Releases page here in future.
